docs/releases/v1.12.0.md

```diff
@@ -0,0 +1,44 @@
+# Disaster recovery Core Module Release 1.12.0
+
+Welcome to the latest release of the `DR` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by team SIGHUP.
+
+This latest release upgrades the components in the module to its latest stable release along with adding support for Kubernetes release `v1.26`.
+
+## Component Images 🚢
+
+| Component                           | Supported Version                                                                                 | Previous Version |
+|-------------------------------------|---------------------------------------------------------------------------------------------------|------------------|
+| `velero`                            | [`v1.11.0`](https://github.com/vmware-tanzu/velero/releases/tag/v1.11.0)                          | `1.10.1`          |
+| `velero-plugin-for-aws`             | [`v1.7.0`](https://github.com/vmware-tanzu/velero-plugin-for-aws/releases/tag/v1.7.0)             | `1.6.1`          |
+| `velero-plugin-for-microsoft-azure` | [`v1.7.0`](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/releases/tag/v1.7.0) | `1.6.1`          |
+| `velero-plugin-for-gcp`             | [`v1.7.0`](https://github.com/vmware-tanzu/velero-plugin-for-gcp/releases/tag/v1.7.0)             | `1.6.1`          |
+| `velero-plugin-for-csi`             | [`v0.5.0`](https://github.com/vmware-tanzu/velero-plugin-for-csi/releases/tag/v0.5.0)             | `0.4.1`          |
+
+> Please refer to the individual release notes to get a detailed information on each release.
+
+## Features 💥
+
+- Updated Velero to v1.11.0
+- Updated plugins
+- Added support for Kubernetes v1.26.x
+
+## Update Guide 🦮
+
+### Process
+
+To upgrade this core module from `v1.11.x` to `v1.12.0`, you need to download this new version, then apply the `kustomize` project.
+
+```bash
+kubectl apply -f katalog/velero-base/crds.yaml
+kustomize build katalog/velero-on-prem | kubectl apply -f -
+# or
+kustomize build katalog/velero-aws | kubectl apply -f -
+# or
+kustomize build katalog/velero-gcp | kubectl apply -f -
+# or
+kustomize build katalog/velero-azure | kubectl apply -f -
+# and/or
+kustomize build katalog/velero-restic | kubectl apply -f -
+```
+
+For additional info you can have a look [here](https://velero.io/docs/v1.11/upgrade-to-1.11/)
```

Comment:
Let's add something about the terraform requirement update

Reply:
Done!